wordpress-mobile/WordPress-iOS
# Stories: Add Media Picker Integration

* Adds combined Media Picker with Photos + WordPress media in a Tab Bar controller
* Integrates combined Media Picker with Kanvas for Stories

Related Kanvas API update: https://github.com/tumblr/kanvas-ios/pull/90

| Photos | Media |
| ------- | ------ |
| <a href="https://user-images.githubusercontent.com/3250/108586767-b433e100-730d-11eb-8fc1-085fcc78cabf.png"><img src="https://user-images.githubusercontent.com/3250/108586767-b433e100-730d-11eb-8fc1-085fcc78cabf.png" width="300"></a>| <a href="https://user-images.githubusercontent.com/3250/108586764-af6f2d00-730d-11eb-830c-014ffbf16a32.png"><img src="https://user-images.githubusercontent.com/3250/108586764-af6f2d00-730d-11eb-830c-014ffbf16a32.png" width="300"></a> | 


## Testing

* Open the Blog Details page
* Tap the FAB
* Select "Story post"
* Tap on the Media Picker button in the lower right
* Make sure that media from device and WordPress is properly loaded into the editor

PR submission checklist:
